docs/application-connector/docs/01-01-application-connector.md

```diff
@@ -16,6 +16,6 @@ The AC gives you this functionality:
 - Allows to map an App to a Kyma Namespace and use its registered APIs and Event catalogs in the context of that Namespace.
 - Integrates the registered APIs and Event catalogs with the Kyma Service Catalog.
 
-All of the AC components scale independently, which allows to adjust it to fit the needs of the implementation built using Kyma.
+The scaling of all AC components is independent and automatic, which allows to adjust it to fit the needs of the implementation built using Kyma.
```

Comment:
I do not think that automatic scaling is enabled by default.

Reply:
reversed to previous version (more accurate)